store/sql_store.go

```diff
@@ -110,6 +111,14 @@ func initConnection() *SqlStore {
 		}
 	}
 
+	if len(utils.Cfg.SqlSettings.DataSourceSearchReplica) == 0 {
+		sqlStore.searchReplica = sqlStore.master
```

Comment:
ooh yeah, good point! I'll change it to an array, as I don't think there's any other way of doing it that isn't excessively complicated.